zebra-chain/src/transaction.rs

```diff
@@ -99,6 +99,21 @@ pub enum Transaction {
         /// The shielded data for this transaction, if any.
         shielded_data: Option<ShieldedData>,
     },
+    /// An Orchard transaction (`version = 5`).
```

Comment:
Terminology: I am unsure if this is an Orchard transaction or a NU5 transaction.

Reply:
It is just a version 5 transaction. (The numbering is a coincidence.)